src/wallet/transaction.h

```diff
@@ -48,7 +48,12 @@ struct TxStateBlockConflicted {
     int conflicting_block_height;
 
     explicit TxStateBlockConflicted(const uint256& block_hash, int height) : conflicting_block_hash(block_hash), conflicting_block_height(height) {}
-    std::string toString() const { return strprintf("Conflicted (block=%s, height=%i)", conflicting_block_hash.ToString(), conflicting_block_height); }
+    std::string toString() const { return strprintf("BlockConflicted (block=%s, height=%i)", conflicting_block_hash.ToString(), conflicting_block_height); }
+};
+
+//! State of transaction with one or more mempool conflicts.
+struct TxStateMempoolConflicted {
+    std::string toString() const { return strprintf("MempoolConflicted"); }
```

Comment:
In  a6ae5b23b1497ab6f4899db49348db623700a2d8 "wallet: track mempool conflicts":

Nit: Should this perhaps also inform about the conflicting transactions like the _BlockConflicted_ state informs about the block?

Reply:
This could be done in a follow-up if people want it, because currently the conflicting transactions are stored in the `CWalletTx`, and not the transaction state, so this function is not aware of the txids of the mempool conflicts.